doc/part-1-github-ui/workflows-and-best-practices.md

```diff
@@ -26,6 +26,10 @@ Workflow common practices and best practices
 
 ## Tasks
 
+You will now learn to use everything you have learnt about in the learning videos. You should not have any issues while working on this task, if you follow the instructions to the latter. But incase you run into any issues, feel free to ask for help in the [Github discussion for part 1.](https://github.com/Ifycode/git-github-training/discussions/79) Or you can send the pull request anyway, and state the issue you have there in your pull request.
+
+**Task**: Follow the instructions in the issue created from the template in the tasks section of the [github contributions page.](https://obiagba-mary.gitbook.io/git-and-github-training/part-1-github-ui/github-contributions). This time, the you are to interact with the develop branch as shown in the learning videos.
+
```

Comment:
As for this PR, nice job adding the correct content. But you did not add the hint too. Add the hint also. You can read the related issue again to confirm.

Reply:
Made changes please review

Reply:
I am made another change please review